src/metabase/public_settings/premium_features.clj

```diff
@@ -376,6 +376,10 @@
   ;; everything else.
   :getter #(has-feature? :embedding))
 
+(define-premium-feature enable-embedding-sdk-origins?
```

Comment:
the `enable-embedding-sdk` defsetting will make 2 functions: One called `enable-embedding-sdk`, and `enable-embedding-sdk!`. So I don't think `enable-embedding-sdk?` will actually collide.

Reply:
That's interesting. Maybe it's something else that makes `enable-embedding-sdk?`? I couldn't really name it like that here since BE won't start for me.